scripts/irods/test/test_all_rules.py

```diff
@@ -489,6 +486,46 @@ def test_writeLine_config_first_3477(self):
             IrodsController().start()
 
 
+    @unittest.skipUnless(plugin_name == 'irods_rule_engine_plugin-irods_rule_language', 'only applicable for irods_rule_language REP')
+    def test_msiServerMonPerf_default_3736(self):
+        path_to_irodsServerMonPerf = os.path.join(paths.home_directory(), 'msiExecCmd_bin', 'irodsServerMonPerf')
+        path_to_irodsServerMonPerf_template = path_to_irodsServerMonPerf + '.template'
+        irodsServerMonPerf_already_existed = os.path.exists(path_to_irodsServerMonPerf)
+
+        rule_file="test_msiServerMonPerf.r"
+        rule_string= '''
+msiTestServerMonPerf {{
+    msiServerMonPerf("verbose", "default");
```

Comment:
The test rule file in the icommands repo being replaced by this test runs it both ways, so we probably should here too.

```suggestion
    msiServerMonPerf("default", "default");
    msiServerMonPerf("verbose", "default");
```

Reply:
Also a good point. Done.